CONTRIBUTING.md

```diff
@@ -106,14 +106,14 @@ exports.config = {
 ```
 
 #### Running Tests
-If you've written a new provider you are an awesome person. If you now write tests for it, you are even more awesome. And who doesn't want to be more awesome right?
+If you've written a new provider you are an awesome person. If you now write tests for it, you are even more awesome. And who doesn't want to be more awesome, right?
 
 #### Codestyle
-I'm using Eslint to maintain quote style and quality. Do not skip it...
+I'm using ESLint to maintain quote style and quality. Do not skip it...
 
-##### To do before merging:
+##### To-do before merging:
 
-- executed tests? (`pnpm test`)
```

Comment:
See https://github.com/orangecoding/fredy/pull/144/files#r2223976502

Reply:
interesting. That was a leftover which I forgot to remove I assume